scripts/build.sh

```diff
@@ -28,6 +28,9 @@ cd ${SCRIPT_ROOT}
 echo "Building core image..."
 docker build -t ${PREFIX}/vizier-core -f ${CMD_PREFIX}/manager/Dockerfile .
 
+echo "Building REST API for core image..."
+docker build -t ${PREFIX}/vizier-core-rest -f ${CMD_PREFIX}/manager_rest/Dockerfile .
```

Comment:
build the image for manager REST API. it will be a separate service running in kubernetes.